app/views/RoomView/UploadProgress.js

```diff
@@ -157,7 +158,7 @@ export default class UploadProgress extends Component {
 						<Text style={styles.tryAgainButtonText}>{I18n.t('Try_again')}</Text>
 					</TouchableOpacity>
 				</View>
-				<Icon name='close' size={20} color='#9EA2A8' onPress={() => this.deleteUpload(item)} />
+				<Icon name='close' size={20} color='#9EA2A8' onPress={() => this.deleteUpload(this.uploads)} />
```

Comment:
Calling `deleteUpload` with `this.uploads` would remove all uploads instead of just the tapped item.

Reply:
I see. What would be for just the item? Because the `item` is not a valid object.